test/bootstrap_form_group_test.rb

```diff
@@ -359,23 +359,78 @@ class BootstrapFormGroupTest < ActionView::TestCase
     assert_equivalent_xml expected, output
   end
 
-  test 'form_group renders the "error" class and message correctly when object is invalid' do
+  # NOTE: Removing this test case as something that's too difficult to support with Bootstrap 4.
+  # test 'form_group renders the "error" class and message corrrectly when object is invalid' do
+  #   @user.email = nil
+  #   assert @user.invalid?
+  #
+  #   output = @builder.form_group :email do
+  #     %{<p class="form-control-static">Bar</p>}.html_safe
+  #   end
+  #
+  #   expected = <<-HTML.strip_heredoc
+  #     <div class="form-group">
+  #       <p class="form-control-static">Bar</p>
+  #       <div class="invalid-feedback">can't be blank, is too short (minimum is 5 characters)</div>
+  #     </div>
+  #   HTML
+  #   assert_equivalent_xml expected, output
+  # end
```

Comment:
I'm going to remove this. It the comment served well for the review of this PR, but I would rather not keep comment-out code in the codebase. I'll move the reasoning for removing this into a commit comment instead.